test/helpers/kubectl.go

```diff
@@ -878,9 +877,28 @@ func (kub *Kubectl) CiliumPolicyAction(namespace, filepath string, action Resour
 	err = WithTimeout(
 		body,
 		"cannot change state of resource correctly; command timed out",
-		&TimeoutConfig{Timeout: timeout})
+		&TimeoutConfig{Timeout: 1})
```

Comment:
What is the need for this change? This is unrelated to the content in the commit message, apart from that it utilizes the new SSHMetaError. Factor it out into a separate commit and explain why the change is necessary in the commit message.

Reply:
ouch, embarrassing, my bad!